### DISCUSSION TITLE
Mini-dashboard improvements

### DISCUSSION THREAD

**Discussion by intoxicated** [meilisearch/product#637] [Feedback & Feature Proposal]
Hi meili team! ✋ 

I've successfully migrated from algolia to meilisearch (there are a couple of feature parity such as delete by filter and partial update array attribute but was able to workaround). And I occasionally needed to add/update/remove entries in dashboard. So I would like to see some minor improvements in dashboard.

1. functionality to add / remove / update entry (or batch) 
2. search with filter (facets)

**Comment by gmourier** [comment]
Hey @intoxicated 👋

Thanks for the feedback. 

For now, it's not a priority to invest in the evolution of the mini-dashboard. I'm keeping the discussion open so that others can join in.

In the meantime, [you may want to try out some dashboards made by the community](https://github.com/meilisearch/awesome-meilisearch#community-tools); some may have the improvement you request here!

**Comment by CaroFG** [comment]
Hi! Priorities have changed and we want to hear all your ideas:

- What's working well and what isn't?
- What improvements would you like to see?

**Comment by Strift** [comment]
We opened a RFC dedicated to the upcoming revamp of the search preview.

Please contribute to the discussion here: [RFC: Improving the search preview](https://github.com/orgs/meilisearch/discussions/766)
